src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -749,7 +749,8 @@ void readAnalyze(TrackPointer track, double sampleRate, int timingOffset, bool i
 
     rekordbox_anlz_t anlz = rekordbox_anlz_t(&ks);
 
-    double sampleRateFrames = sampleRate * 2.0;
+    double sampleRateScaled = sampleRate / 1000.0;
+    double sampleRateFrames = sampleRateScaled * 2.0;
```

Comment:
If you multiply by the number of channels = 2 then this no longer denotes *frames* of an interleaved signal, but individual *samples*. Also the number 2 should be stored in a constant `kNumberOfChannels` or `kChannelCount` to document this assumption.

Reply:
CHnaged to mixxx::kEngineChannelCount